src/functions/functionalConfig.ts

```diff
@@ -16,6 +16,12 @@ export const functionConfig: { [key: string]: NonEmptyArray<IApiAccess> } = {
       path: "reference/*",
     },
   ],
+  "TechRecord.Create": [
+    {
+      verbs: ["POST", "OPTIONS"],
+      path: "vehicles/*",
```

Comment:
I don't think this needs the `/*` on the end as the POST endpoint is the base endpoint no?

Reply:
Agree apologies